From b65062a0adb95cb9203086fe2b47e529551cb601 Mon Sep 17 00:00:00 2001
From: Ananth Krishna R <ananth.krishna.r@intel.com>
Date: Thu, 18 Dec 2014 18:39:47 +0530
Subject: [PATCH] [cpufreq_interactive] Fix compilation warnings

This patches brings in the neccessary changes to fix the compilation
warnings seen after enabling touch boost functionality in interactive
governor.

Change-Id: Ie84082cb6890b23ebb339bf14598262e46d855f4
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-13909
Signed-off-by: Ananth Krishna R <ananth.krishna.r@intel.com>
Reviewed-on: https://android.intel.com:443/312895
---
 drivers/cpufreq/cpufreq_interactive.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/cpufreq/cpufreq_interactive.c	2015-01-28 16:49:02.098395237 -0800
+++ b/drivers/cpufreq/cpufreq_interactive.c	2015-01-28 16:49:06.170395371 -0800
@@ -1367,12 +1367,15 @@ static struct attribute *interactive_att
 	&above_hispeed_delay_gov_pol.attr,
 	&hispeed_freq_gov_pol.attr,
 	&go_hispeed_load_gov_pol.attr,
+	&touchboost_freq_gov_pol.attr,
 	&min_sample_time_gov_pol.attr,
 	&timer_rate_gov_pol.attr,
 	&timer_slack_gov_pol.attr,
 	&boost_gov_pol.attr,
 	&boostpulse_gov_pol.attr,
 	&boostpulse_duration_gov_pol.attr,
+	&touchboostpulse_gov_pol.attr,
+	&touchboostpulse_duration_gov_pol.attr,
 	&io_is_busy_gov_pol.attr,
 #ifdef CONFIG_IRQ_TIME_ACCOUNTING
 	&irq_load_threshold_gov_pol.attr,
